src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt

```diff
@@ -268,6 +272,11 @@ private fun SpacingContext.blockMustBeMultiLine(): Boolean {
     }
 }
 
+private fun SpacingContext.needsBlankLineBetweenItems(): Boolean {
+    // Allow to keep consecutive runs of `use`, `const` or `static` items without blank lines
+    return !(elementType1 == elementType2 && elementType1 in ONE_LINE_ITEMS)
```

Comment:
Hmm though I think that this counterexample might not be counterexample at all. How would you like to format this thing?

```rust
mod foo;
//! Where is your God now?
mod bar;
```

or

```rust
mod foo;

//! Where is your God now?

mod bar;
```